rest/data/pod.go

```diff
@@ -1,14 +1,29 @@
 package data
 
 import (
+	"fmt"
+	"net/http"
+
 	"github.com/evergreen-ci/evergreen/model/pod"
+	"github.com/evergreen-ci/gimlet"
 	"github.com/pkg/errors"
 )
 
 // DBPodConnector implements the pod-related methods from the connector via
 // interactions with the database.
 type DBPodConnector struct{}
 
+// CreatePod inserts the given pod.Pod.
+func (c *DBPodConnector) CreatePod(pod *pod.Pod) error {
```

Comment:
Can you write a test for the data connector? Usually, we write tests for each layer of abstraction. In this case, the REST route and the data connector are separate abstractions dealing with different concerns.